### ISSUE TITLE
PSNR is lower than the value in the paper

### ISSUE THREAD

**Issue by CoffinmerchantNewton** [Baisonm-Li/HSR-KAN#4]
Hello, I would like to reproduce this part of your work, but the PSNR value of the CAVE dataset X4, which I processed using the HSR files in your warehouse, reached 39.9% at 200 epochs and has remained at this value without further improvement Please tell me how to solve this problem

**Comment by baiSongL** [comment]
The report in the article indicates that the model was trained for 1000 epochs, with the first 200 epochs not achieving full convergence. Additionally, to ensure the generalization capability of the KAN network, a sparse loss function was manually implemented.
